docs/how_tos/i18n.rst

```diff
@@ -236,6 +236,56 @@ Load up your translation files
 
 #. As of this writing, ``frontend-platform/i18n`` reads the locale from the user language preference cookie, or, if none is found, from the browser's language setting. You can verify everything is working by changing your language preference in your account settings. If you are not logged in, you can change your browser language to one of the languages you have translations for.
 
+
+********************
+Migrating to react-intl@5
```

Comment:
I think one more addition here might be related to [testing](https://formatjs.io/docs/react-intl/upgrade-guide-3x/#testing). Looks like `IntlProvider.getChildContext()` is [used](https://github.com/search?q=org%3Aopenedx+getChildContext&type=code) in a few places.

Reply:
Nice catch! I've completely missed it 😞 

added it to the docs, thanks!